common/src/mailbox_api.rs

```diff
@@ -293,6 +294,45 @@ impl Default for InvokeDpeReq {
     }
 }
 
+// EXTEND_PCR
+#[repr(C)]
+#[derive(Debug, AsBytes, FromBytes, PartialEq, Eq)]
+pub struct ExtendPcrReq {
+    pub hdr: MailboxReqHeader,
+    pub pcr_idx: u32,
+    pub data_size: u32,
+    pub data: [u8; ExtendPcrReq::DATA_MAX_SIZE],
+    _pad: [u8; 4 - (ExtendPcrReq::DATA_MAX_SIZE % 4) % 4],
```

Comment:
I think extending PCR is typically only done with a digest which has the size of the PCR reg itself?